WinUIGallery/ControlPages/ContentDialogPage.xaml.cs

```diff
@@ -37,6 +37,22 @@ private async void ShowDialog_Click(object sender, RoutedEventArgs e)
             dialog.Content = new ContentDialogContent();
             dialog.RequestedTheme = (VisualTreeHelper.GetParent(sender as Button) as StackPanel).ActualTheme;
 
+            dialog.Loaded += (s, args) =>
+            {
+                // Apply a dark overlay to the title bar of the window while the dialog is open
```

Comment:
Can you update the comment to include that this workaround for the custom titlebar? In the meantime, I'll open a bug internally on WinUI.